torch/_inductor/codecache.py

```diff
@@ -877,9 +877,16 @@ def get_include_and_linking_paths(
         # (later on, we copy the include paths from cpp_extensions into our remote dir)
         ipaths.append("include")
 
+    static_link_libs = ""
+    if aot_mode and config.is_fbcode():
+        # For Meta internal cuda-12, it is recommended to static link cudarrt
+        static_link_libs = " ".join(
```

Comment:
might be marginally cleaner to keep this as a list here (instead of joining) and then join with libs below like:
```
libs = " ".join(static_link_libs + ["-l" + p for p in libs])
```

Reply:
Really nbd either way though